### DISCUSSION TITLE
ImageFrame shop

### DISCUSSION THREAD

**Discussion by ReiTheGoat01** [Stardust-Labs-MC/StardustMC#36] [Ideas]
With donators getting access to one custom image frame they of course want to use it for something memorable and unique. So what if there was a way to obtain more generic or utility images for builds.
I propose an extension to the adminshop dedicated to these simple ImageFrames for all players to buy. These could include maps of the stardust logo and maps with block textures on them. 

Using block texture images and invisible item frames you can unlock a wide variety of building techniques. For example, one way glass, seamless walk-through walls, and cover up light blocks in builds.

**Comment by aisukuma** [comment]
[Separate Idea but relating to custom images from Rei (requires joining the Stardust Labs discord)](https://discord.com/channels/738046951236567162/1131733013693218926/1147720932048252958): 

> Kuma what are thoughts on having imageframe requests/music disc requests be given out as vouchers during events on the server as rare rewards from a minigame tournaments or something

**Reply by aisukuma** [reply]
Mayhaps. I'll forward this to catter to see what he thinks

**Comment by aisukuma** [comment]
> Using block texture images and invisible item frames you can unlock a wide variety of building techniques. For example, one way glass, seamless walk-through walls, and cover up light blocks in builds.

Fun fact, check the Stardust Labs Tower in the interrogation room where "one-way glass" is implemented

But yes, I do plan on adding an extension to sell custom images

**Comment by aisukuma** [comment]
Vouchers for ImageFrames will be implemented, will need to brainstorm for music discs as I'm changing the system for that
